### PR TITLE
Add visualization

### PR DESCRIPTION
**Pull request by hkzhang95**
Add visualization for test.py and update README.md

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -168,4 +171,21 @@ def load_model(model, pretrained_path, load_to_cpu):
...
+
+        # show image
+        if args.show_image:
+            for b in dets:
+                if b[4] < args.vis_thres:
+                    continue
+                text = "{:.4f}".format(b[4])
+                b = list(map(int, b))
+                cv2.rectangle(img_raw, (b[0], b[1]), (b[2], b[3]), (0, 0, 255), 2)
+                # cx = b[0] + (b[2] - b[0]) / 2 - 5
```
Comment (zisianw) [inline test.py]: We could remove this line.

File: test.py [hunk 2]
```diff
@@ -168,4 +171,21 @@ def load_model(model, pretrained_path, load_to_cpu):
...
+                cx = b[0]
+                cy = b[1] + 12
+                cv2.putText(img_raw, text, (int(cx), int(cy)),
```
Comment (zisianw) [inline test.py]: We could drop the int().

**Review by zisianw — CHANGES_REQUESTED** [review]
The update looks good! I've left a few comments before this gets merged. :smile:

**Comment by hkzhang95** [comment]
Thanks for your comments, I have fixed the redundancy.

**Comment by zisianw** [comment]
All good, thanks! :smiley:

**Pull request merged**